packages/salesforcedx-vscode-soql/src/queryDataView/queryDataFileService.ts

```diff
@@ -91,8 +84,11 @@ export class QueryDataFileService {
 
   private showSaveSuccessMessage(savedFileName: string) {
     vscode.window.showInformationMessage(
-      // TODO: i18n
-      `The results were saved in the ${QUERY_RESULTS_DIR_NAME} folder as: ${savedFileName}`
+      nls.localize(
+        'info_file_save_success',
+        QUERY_RESULTS_DIR_NAME,
+        savedFileName
+      )
```

Comment:
<img width="474" alt="save-results-success" src="https://user-images.githubusercontent.com/599418/101827038-075b0d80-3aed-11eb-8998-ad775c123925.png">

Reply:
or We saved the results in the query-folder as:

Reply:
Updated.